frontend/src/screens/internal-apps/ZapPlanner.tsx

```diff
@@ -87,35 +94,134 @@ export function ZapPlanner() {
   const [amount, setAmount] = React.useState("");
   const [comment, setComment] = React.useState("");
   const [senderName, setSenderName] = React.useState("");
+  const [frequencyValue, setFrequencyValue] = React.useState("1");
+  const [frequencyUnit, setFrequencyUnit] = React.useState("months");
+  const [amountCurrency, setAmountCurrency] = React.useState<"BTC" | "USD">(
+    "USD"
+  );
+  const [currencies, setCurrencies] = React.useState<string[]>(["BTC"]);
+
+  const [convertedAmount, setConvertedAmount] = React.useState<string>("");
+  const [satoshiAmount, setSatoshiAmount] = React.useState<number | undefined>(
+    undefined
+  );
+
+  React.useEffect(() => {
+    // fetch the fiat list and prepend sats/BTC
+    async function fetchCurrencies() {
+      try {
+        const res = await fetch("https://getalby.com/api/rates");
+        const data: Record<string, { name: string }> = await res.json();
+        const fiatCodes = Object.keys(data)
+          .map((c) => c.toUpperCase())
+          // drop "BTC" so it only lives in our prepended slot
```

Comment:
I don't think our API is supposed to return BTC. So we might be able to remove this line